### PR TITLE
external on isliquidationDeadlinePassed

### PR DESCRIPTION
**Pull request by jacko125**
Picked up on Slither report

### CODE REVIEW

File: contracts/Liquidations.sol [hunk 1]
```diff
@@ -124,7 +124,7 @@ contract Liquidations is Owned, MixinResolver, ILiquidations {
         return false;
     }
 
-    function isliquidationDeadlinePassed(address account) public view returns (bool) {
+    function isliquidationDeadlinePassed(address account) external view returns (bool) {
```
Comment (justinjmoses) [inline contracts/Liquidations.sol:127]: ahhh, it should be `isLiquidationDeadlinePassed` (camelCase)

Comment (jacko125) [reply contracts/Liquidations.sol:127]: those weak hands

Comment (jacko125) [reply contracts/Liquidations.sol]: fixed

Comment (justinjmoses) [reply contracts/Liquidations.sol]: thanks. :pray:

the tests need fixing as well

**Pull request merged**